src/software/backend/output/radio/mrf/messages.h

```diff
@@ -205,4 +205,6 @@ namespace MRF
         "Second dongle on this channel+PAN";
     static constexpr const char* TRANSMIT_QUEUE_FULL_MESSAGE = "Transmit Queue Full";
     static constexpr const char* RECEIVE_QUEUE_FULL_MESSAGE  = "Receive Queue Full";
+    static constexpr const char* PACKET_ABOVE_MAX_SIZE =
```

Comment:
if we want to see this from the software side (i.e. visualizer robot status table, post sensorfusion), then we would need to have an equivalent in the ErrorCode enum in TbotsRobotMsg

Reply:
So this isn't a robot status, the annunciator is a different system specifically for the dongle.